draft-ietf-rats-architecture.md

```diff
@@ -1317,3 +1317,160 @@ use beyond the period for which it deems the Attestation Result to remain
 valid.  Thus, if the Attestation Result sends a validity lifetime
 in terms of time(RX)-time(RG), then the Relying Party can check
 `time(OP) - time(ER) < time(RX)-time(RG)`.
+
+# Appendix B: System Examples
```

Comment:
I think that by "System", you mean something specific, yet the text is rather general.  I suggest making the example even more specific: pick one of Smartphone. Desktop PC. VoIP phone. Smart IoT Speaker.

Reply:
By "system" I mean an end-end set of entities that make up a complete function system with nothing left out. The suggestion of Smartphone doesn't work because it is only a small part of the "system". I'm OK with a different word, perhaps "End-to-end Complete Examples"?